edxval/pacts/utils.py

```diff
@@ -24,66 +23,18 @@ def clear_database():
     VideoTranscript.objects.filter(video__edx_video_id=VIDEO_ID).delete()
 
 
-def setup_successful_video_status_state():
-    """
-    Setup provider state for a successful video status update.
-    """
-    remove_view_auth_and_perms(VideoStatusView)
-    create_video()
-
-
-def setup_unsuccessful_video_status_state():
-    """
-    Setup the provider state for unsuccessful video status update.
-    """
-    remove_view_auth_and_perms(VideoStatusView)
-
-
-def setup_successful_video_image_state():
-    """
-    Setup the provider state for successful video image update.
-    """
-    remove_view_auth_and_perms(VideoImagesView)
-    create_course_video()
-    create_profiles()
-
-
-def setup_unsuccessful_video_image_state():
-    """
-    Setup the provider state for unsuccessful video image update.
-    """
-    remove_view_auth_and_perms(VideoImagesView)
-
-
 def setup_successful_video_details_state():
     """
     Setup the provider state for successful video details update call.
     """
-    remove_view_auth_and_perms(VideoDetail)
```

Comment:
Q: Instead of this, I see we have a new middleware now, how its bypassing the auth?

Reply:
middleware is setting request.user to be an authenticated user instead of anonymous user.